src/osd/OSD.cc

```diff
@@ -9805,7 +9805,6 @@ int OSD::init_op_flags(OpRequestRef& op)
             (iter->op.op != CEPH_OSD_OP_DELETE) &&
             (iter->op.op != CEPH_OSD_OP_SETXATTR) &&
             (iter->op.op != CEPH_OSD_OP_RMXATTR) &&
-            (iter->op.op != CEPH_OSD_OP_STARTSYNC) &&
```

Comment:
we need to keep this for every op the kernel client sends will force a promotion